src/amazons3/Module.md

```diff
@@ -2,12 +2,13 @@ Connects to Amazon S3 from Ballerina.
 
 # Module Overview
 
-The Amazon S3 client allows you to access the Amazon S3 REST API through ballerina. The following section provide you the details on connector operations.
-
+The Amazon S3 client allows you to access Amazon S3 REST API through ballerina. The following section provide you the 
+details on connector operations.
 
 **Buckets Operations**
 
-The `wso2/amazons3` module contains operations that work with buckets. You can list the existing buckets, create a bucket, delete a bucket and list objects in a bucket.
+The `wso2/amazons3` module contains operations that work with buckets. You can list the existing buckets, create a bucket, 
+delete a bucket and list objects in a bucket.
```

Comment:
```suggestion
delete a bucket, and list objects in a bucket.
```

Added serial comma. Normally we have a comma before "and" in a list.

Reply:
Fixed